#161342804 Create article functionality

victor-nweze-deel opened the pull request:
#### What does this PR do?
- Enables a user create articles

#### Description of Task to be completed?
- Create component for creating articles
- Incorporate tinymce WYSIWYG editor
- Configure tinymce to automatically upload images to cloudinary

#### How should this be manually tested?
- Git clone this repository
- Cd into the haven-ah-frontend folder
- Run `npm run start:dev`
- Hit the create article button from the landing page

#### What are the relevant pivotal tracker stories?
- [#161342804](https://www.pivotaltracker.com/story/show/161342804)

#### Screenshots (if appropriate)
<img width="1202" alt="screen shot 2018-12-11 at 2 19 03 pm" src="https://user-images.githubusercontent.com/41582565/49803348-d144f080-fd4f-11e8-812e-72e72c1190b4.png">

ucheg6 commented:
Good work bro @ProfJigsaw 👍

TheoOkafor commented on src/components/containers/createArticle/createArticle.jsx:
```diff
@@ -0,0 +1,252 @@
+                const formData = new FormData();
+                formData.append('upload_preset', 'default_preset');
+                formData.append('file', blobInfo.blob(), blobInfo.filename());
+                axios({
```
Nice

TheoOkafor reviewed (commented):
Nice work @ProfJigsaw 
👍

seyi-adeleke commented on src/components/views/CreateArticlePage.jsx:
```diff
@@ -0,0 +1,18 @@
+    <div className="layout">
+      <div className="container">
+        <header>
+          <Navbar isLoggedIn />
```
why are you hardcoding isLoggedIn ?

victor-nweze-deel replied:
Yeah, Theo had that. I was waiting for his branch to be merged in.

victor-nweze-deel committed:
ft(articles): Create articles

- Create component for creating articles
- Incorporate tinymce WYSIWYG editor
- Configure tinymce to automatically upload images to cloudinary
